### DISCUSSION TITLE
import images to device_types via API

### DISCUSSION THREAD

**Discussion by nyevess** [netbox-community/netbox#6624] [Q&A]
Hello!
I have a lot of problems to import images to device_types:
What information is needed in the field "content_type"?
I tried with:
extras.ImageAttachments
extras.image-attachments
But I can't found information about this in the documentation.
And the other question is, what information is needed in the field "image"?
I need put the path of my PC? or how?
Thank you in advance!
<img width="166" alt="2021-06-17 17_12_59-NetBox API" src="https://user-images.githubusercontent.com/69692460/122425837-11743200-cf90-11eb-8405-ee220b8e19dc.png">

**Comment by drygdryg** [comment]
See https://github.com/netbox-community/netbox/discussions/6845#discussioncomment-1938587
